src/pages/rooms/[id]/questions/[questionId]/answers/RangeAnswersResults.tsx

```diff
@@ -0,0 +1,62 @@
+// wie stelle ich die Zahlen von 1 - 10 als default da?
+
+import React from 'react';
+import Answer from '../../../../../../types/answer';
+// import AnswerCreate from '../../../../../../types/answer';
+
+interface Props {
+  answers: Answer[];
+}
+
+export default function RangeAnswersResults(props: Props) {
```

Comment:
Just tried to do that but failed. Wanted to add a boolean key rangeOrFix to the form component. From there it would be passed on to all the child components. However I am unsure, whether rangeOrFix needs to be defined in the backend.